parts/linux/cloud-init/artifacts/kubelet.service

```diff
@@ -18,9 +18,6 @@ ExecStartPre=/bin/mount --make-shared /var/lib/kubelet
 ExecStartPre=-/sbin/ebtables -t nat --list
 ExecStartPre=-/sbin/iptables -t nat --numeric --list
 
-{{/* This is a workaround to setup azure0 bridge for CNI */}}
-ExecStartPre=/usr/local/bin/configure_azure0.sh
```

Comment:
after making these changes I think you also need to regenerate files by running make generate

Reply:
Looks like you already did that so lets just use yours. https://github.com/Azure/AgentBaker/pull/512